### PR TITLE
add fedora spec file and cmake changes

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: opae.spec [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+         board_vc \
+         fpgaconf \
+         fpgainfo \
+         fpgametrics \
+         userclk \
+         hello_fpga \
+         hello_events \
+         object_api \
+         mmlink  \
+         opae-c-ase \
```
Comment (tswhison) [inline opae.spec]: Please exclude opae-c-ase.

File: opae.spec [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+
+
+mkdir -p %{buildroot}%{_sysconfdir}/systemd/system/
+
+%clean
+
+%post
+mkdir -p %{_sysconfdir}/ld.so.conf.d
+echo "" > %{_sysconfdir}/ld.so.conf.d/opae-c.conf
```
Comment (tswhison) [inline opae.spec]: Why do we create this file?  It's empty.

File: opae.spec (hunk 1 above)
Comment (anandaravuri) [reply opae.spec]: fixed

File: opae.spec (hunk 2 above)
Comment (anandaravuri) [reply opae.spec]: Remove create file code
Fixed

**Pull request merged**